scripts/cti

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Copyright (c) The Libra Core Contributors
+# SPDX-License-Identifier: Apache-2.0
+set -e
+
+aws codebuild list-projects >/dev/null || (echo "Failed to access codebuild, try awsmfa?"; exit 1)
+ssh ssh.cluster-test-ci.aws.hlw3truzy4ls.com echo "ssh ok" >/dev/null || (echo "Failed to ssh to cluster test instance. Try renew corp canal cert with cc-certs"; exit 1)
+
+PR=""
+
+while (( "$#" )); do
+  case "$1" in
+    -p|--pr)
+      PR=$2
+      shift 2
+      ;;
```

Comment:
It would be good to add a `--help` flag which prints the usage for this tool similar to https://github.com/libra/libra/blob/master/docker/build-aws-base.sh#L37

Reply:
Strictly speaking if you run `cti --help` you will see exactly this message :)